trade-finance-manager-ui/server/controllers/case/index.ff-test.js

```diff
@@ -1,13 +1,8 @@
-import { AMENDMENT_STATUS, isTfmFacilityEndDateFeatureFlagEnabled } from '@ukef/dtfs2-common';
+import { AMENDMENT_STATUS } from '@ukef/dtfs2-common';
 import caseController from '.';
 import api from '../../api';
 import { mockRes } from '../../test-mocks';
 
-jest.mock('@ukef/dtfs2-common', () => ({
-  ...jest.requireActual('@ukef/dtfs2-common'),
-  isTfmFacilityEndDateFeatureFlagEnabled: jest.fn(),
-}));
```

Comment:
Removed this mock as I don't think it's needed now we have the new `ff` file name way of adding feature flag tests

Reply:
NICE!!